### DISCUSSION TITLE
Prevent nonsense handle call

### DISCUSSION THREAD

**Discussion by bodograumann** [Lancetnik/Propan#75] [Ideas]
I just accidentally messed up the order of `broker.handle` and `broker.start`.
So my handler was not actually used.
Would it be a good idea to throw an exception when registering a handler after the broker was started?

I am not sure if there are use-cases where you register additional handlers and restart the broker afterwards though.
How does FastAPI do this?

**Comment by Lancetnik** [comment]
@bodograumann **FastAPI** does nothing at registering a new handler cuz it costs nothing at HTTP framework: you need no establish new connection or consumer, etc.
So, it looks like a good idea to prevent some obvious problems with a `handler` registering after application starting up. But it shouldn't be an exception: I really don't like an idea to block any potential (even rarely) usecase. So, maybe it should be a warning?

**Comment by Lancetnik** [comment]
So, I create an [Issue](https://github.com/Lancetnik/Propan/issues/76) to implement this. If you can help me - it will be great! I am pretty busy with pydanticV2 migration right now 😞
